test/ocm_test.json

```diff
@@ -0,0 +1,10 @@
+{
+    "access_token": "DUMMYVALUE",
```

Comment:
Maybe move this file.i to the package, as this is the file that is being tested (no need to create another folder structure IMO)

Reply:
I thought we follow the golang project structure: https://github.com/golang-standards/project-layout#test
They suggest creating a test directory, but if this is the only test file, you might be right and it would make sense to move the file to the location where it is actually used. 

@ramonbutter  suggested we could just generate the file during the test process. Do you have an opinion about that?

Reply:
I wouldn't want to generate it, but you can use a in-memory fs for it (can send a link in a sec)

Reply:
Ah I think I know what you mean. We can use the new Go Embed library.

Reply:
Never mind, i think this might be the better solution: https://pkg.go.dev/testing/fstest#MapFS

Reply:
exactly https://github.com/openshift/configuration-anomaly-detection/blob/ab181d77a75051b1e8cfbe53b0cdb3389af0ebbc/pkg/pagerduty/pagerduty_test.go#L517-L523

Reply:
This doesn't  work for us.. the problem is that with MapFS I have to pass a file handle instead of a filepath with type string.
So, even when I pass the filepath to the file in the MapFS, the sdkcfg.Location() function call will fail, because it will search on the wrong filesystem.

TBH, I would prefer that upstream fixes this and implements a io.Reader interface for passing config files around.. this would be by far more testable for downstream projects like ours.

Reply:
that's annoying, but then leave it out.  And to explain my thoughts about generating the test config file (and cleaning them up running the test):
I am not a big fan of defining the config twice, once [here](https://github.com/openshift/configuration-anomaly-detection/pull/26/files#diff-f579fc1054cc8232c801eee714549d207c451899358068b5a4757154b8e9c7aaR29) and in the file. It's unclearer how the expectation is reasoned. Would only go commit stuff for complex test data (photos, binaries, etc.).  But this is just my taste, and am ok with it. Lets just continue and merge for the case this PR works

Reply:
Note: i have added a second config file with invalid json and another test case... I think the /test directory should be fine as suggested in the standard golang project structure repository.